packages/core/src/internal-components/overlay/overlay.element.scss

```diff
@@ -36,10 +38,21 @@
 
 :host([_demo-mode]) {
   position: absolute;
+  z-index: 100000;
```

Comment:
This fixes an issue where overlays in the demo blocks were showing up above the interactive overlays.